### DISCUSSION TITLE
Search doesn't work in database navigator

### DISCUSSION THREAD

**Discussion by Liudvikas-vinted** [dbeaver/dbeaver#19352] [Q&A] [answered]
### Description

When I start typing, search never starts on the database navigator panel, always in the right hand side. Same behaviour after using Ctrl-F. Even if I've just clicked an item on the database navigator, search starts on whatever database/table I have open in the right hand side. 

### DBeaver Version

CE 22.0.1.202203181646

### Operating System

Fedora 37

### Database and driver

Irrelevant, happens in UI with different database drivers.

### Steps to reproduce

1. Click on a database in one of connections in Database Navigator
2. Start typing 
Expected: databases/tables would be filtered by whatever I'm typing
What happens: tables/schemas on the right hand side get filtered, Database Navigator is completely unaffected

### Additional context

Using Gnome on Xorg. Similar small UI bugs are common and they seem to not always trigger, sometimes restarting DBeaver helps.

**Comment by E1izabeth** [comment, marked as answer]
You can put the cursor to the search text box on the top of the Database Navigator and start typing. It works this way.

**Comment by Liudvikas-vinted** [comment]
Thanks, didn't know about that one, but the searchbar only (kinda) works on table names - shouldn't it also filter/jump to databases?

**Comment by E1izabeth** [comment]
It has a switcher to define what kind of object you want to filter
![image](https://user-images.githubusercontent.com/28875055/224744023-61e775ee-345e-4b5f-9244-1eb3a43e052b.png)

**Comment by Liudvikas-vinted** [comment]
thank you so much, it works now!
